virttest/utils_stress.py

```diff
@@ -27,8 +27,16 @@ def __init__(self, params, env):
         self.max_vcpu = params.get("vcpu_maxcpus", 32)
         self.ignore_status = params.get("ignore_status", "no") == "yes"
         self.vms = env.get_all_vms()
-        self.events = params.get("stress_events", "reboot").split(',')
-        self.host_events = params.get("host_stress_events", "").split(',')
+        self.events = params.get("stress_events", "reboot")
```

Comment:
we could possibly hit this issue only with `host_stress_events` as we default it to `""` this fix would not be necessary if we default to `reboot` in case of `stress_events`

Reply:
user can assign `stress_events=""` explicitly and it would lead such situation..

Reply:
ah, okay got it.